lib/bill/bill/sat/interface/types.hpp

```diff
@@ -23,7 +23,7 @@ class var_type {
 	constexpr static uint32_t max_value = (std::numeric_limits<uint32_t>::max() >> 1);
 
 public:
-	constexpr var_type(uint32_t var)
+	constexpr var_type(uint32_t var = 0)
```

Comment:
We need to make sure that this is back-ported to bill.

Reply:
Yeah the changes here are exactly the same as my PR to bill.